dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScope.kt

```diff
@@ -42,7 +42,9 @@ internal class RumActionScope(
     private val startedNanos: Long = eventTime.nanoTime
     private var lastInteractionNanos: Long = startedNanos
 
-    internal val attributes: MutableMap<String, Any?> = initialAttributes.toMutableMap()
+    internal val attributes: MutableMap<String, Any?> = initialAttributes.toMutableMap().apply {
+        putAll(GlobalRum.globalAttributes)
```

Comment:
why do we want to make a snapshot of global attributes at the scope creation time? we don't do that for `RumViewScope`, for example.

Reply:
That's mostly because there could be a long time between the moment an action is caught and it is actually sent, and so when an action has duration 0 we want to have those.

Reply:
It actually may be a mistake to not grab them in the Viewscope, I'll add a task to do that